exonum/src/storage/proof_map_index/key.rs

```diff
@@ -274,20 +265,47 @@ impl ::std::fmt::Debug for DBKey {
             bits.push('|');
         }
 
-        f.debug_struct("DBKey")
-            .field("begin", &self.from)
-            .field("end", &self.to)
+        f.debug_struct("ProofPath")
+            .field("start", &self.start())
+            .field("end", &self.end())
             .field("bits", &bits)
             .finish()
     }
 }
 
+impl StorageKey for ProofPath {
+    fn size(&self) -> usize {
+        PROOF_PATH_SIZE
+    }
+
+    fn write(&self, buffer: &mut [u8]) {
```

Comment:
Is it possible to write a `ProofPath` that doesn't start with a 0 position? If no, maybe add `debug_assert_eq!` to check?

Reply:
No, for branches we serialize `ProofPath` with non zero start position.